trunk-recorder/systems/smartnet_parser.cc

```diff
@@ -338,8 +345,15 @@ std::vector<TrunkMessage> SmartnetParser::parse_message(std::string s,
         message.message_type = GRANT;
         message.freq = getfreq(stack[2].cmd, system);
         message.talkgroup = stack[2].full_address;
-        // message.encrypted    = false;
-        // message.emergency    = false;
+        if (stack[3].status >= 8) {
```

Comment:
Is it wise to assume a greater-than or equal to clause in this instance? 

While Moto SmartNet Type 2 likely won't have any additional status-bits added any time soon, this could be a remote-chance problem in the future if more bit types were to be added or amended.

Reply:
Same applies to: https://github.com/robotastic/trunk-recorder/pull/388/files#diff-4a7f416921ca59465ac64f045a1d86f5R241  :)

Reply:
The status nibble is just that, 1 hex digit, part of a grant or continue OSW. Most of the values (all 16 but 2) are known/defined, with the upper half being encrypted and the lower 8 being not encrypted. A `>= 8` test covers this formulation.